src/Paket.Core/Environment.fs

```diff
@@ -63,6 +63,18 @@ module PaketEnv =
             |> Seq.tryFind File.Exists
             |> Option.map (fun f -> DirectoryInfo(Path.GetDirectoryName(f)))
 
+    let ensureNotExists (directory : DirectoryInfo) =
+        match fromRootDirectory directory with
```

Comment:
I only would check that directory is present and dependencies file is not.
Actually for NuGet conversion you even set the lock file to `None` programmatically.